src/main/java/gift/service/product/CategoryService.java

```diff
@@ -0,0 +1,59 @@
+package gift.service.product;
+
+import gift.global.validate.NotFoundException;
+import gift.repository.product.CategoryRepository;
+import gift.service.product.dto.CategoryCommand;
+import gift.service.product.dto.CategoryModel;
+import java.util.List;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public CategoryService(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    public CategoryModel.Info createCategory(CategoryCommand.Register command) {
```

Comment:
여기는 `@Transasctional`이 빠졌네요!
```suggestion
    @Transasctional
    public CategoryModel.Info createCategory(CategoryCommand.Register command) {
```

Reply:
일부러 넣지 않았습니다! rollback이 필요한 부분이 없어서 넣지 않았는데 넣어야 할까요?

Reply:
앗 아래에는 일관되게 넣어주셨는데 여기에는 빠져서 얘기드렸어요ㅎㅎ

Reply:
앗 실제로 그렇네요! 공부하면서 필요없을것 같아서 뺀 부분도 있고, 나중에 필요할 일이 있지 않을까? 라고 생각하면서 실제로 필요없는데 넣은 부분도 있고 확실히 통일성이 없는 것 같습니다! 넣는 방향으로 바꿔야 겠네요 피드백 감사합니다 핳ㅎ